apps/site/next-data/generators/nvmData.mjs

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const latestKnownVersion = 'v0.40.1';
+
+/**
+ * Fetches the latest NVM version
+ * @returns {Promise<`v${string}`>} Latest NVM version
+ */
+export default async function generateNvmData() {
+  return fetch('https://latest.nvm.sh', { redirect: 'manual' })
+    .then(({ headers }) => {
+      const url = headers.get('location');
```

Comment:
I'm afraid that on static exports Next.js will complain about this

Reply:
ok, why would it complain?

Reply:
Ah, I misread; I thought you were trying to get the current environment (route) headers. Whose Next.js would complain as trying to access runtime-specific info.